api/openapi.ts

```diff
@@ -4,6 +4,173 @@
+    };
+  "/admin/reconcile/usage_events/kube_core_v1_pods/{id}": {
+    /**
+     * Retrieves usage for a given gradient resource.
+     * @description Retrieves usage for a given gradient resource
+     */
+    get: operations["admin-reconcile-getGradientComputeUsage"];
+  };
+  "/admin/reconcile/usage_events/machine-disk/{handle}": {
+    /**
+     * Retrieves usage for a given gradient resource.
+     * @description Retrieves usage for a given gradient resource
+     */
+    get: operations["admin-reconcile-getMachineDiskUsage"];
+  };
+  "/admin/reconcile/usage_events/sessions/{handle}": {
+    /**
+     * Retrieves usage for a given session.
+     * @description Retrieves usage for a given session
+     */
+    get: operations["admin-reconcile-getSessionUsage"];
+  };
+  "/admin/reconcile/usage_events/snapshots/{handle}": {
+    /**
+     * Retrieves usage for a given snapshot resource.
+     * @description Retrieves usage for a given snapshot resource
+     */
+    get: operations["admin-reconcile-getSnapshotUsage"];
+  };
+  "/admin/reconcile/usage_events/templates/{handle}": {
+    /**
+     * Retrieves usage for a given template resource.
+     * @description Retrieves usage for a given template resource
+     */
+    get: operations["admin-reconcile-getTemplateUsage"];
+  };
+  "/admin/reservations": {
+    /**
+     * List machine reservations
+     * @description Fetches a list of machine reservations
+     */
+    get: operations["admin-reservations-list"];
+    /**
+     * Create machine reservation
+     * @description Creates a single machine reservation for a team
+     */
+    post: operations["admin-reservations-create"];
+  };
+  "/admin/reservations/{id}": {
+    /**
+     * Delete machine reservation
+     * @description Deletes a single machine reservation by reservation id
+     */
+    delete: operations["admin-reservations-delete"];
+  };
+  "/admin/reservations/{id}/extend": {
+    /**
+     * Delete machine reservation
+     * @description Updates the end date of the reservation
+     */
+    patch: operations["admin-reservations-extend"];
+  };
+  "/admin/reservations/pools": {
+    /**
+     * List machine reservations
+     * @description Fetches the list of pools for reservations.
+     */
+    get: operations["admin-reservations-pools"];
+  };
+  "/admin/teams/{teamId}/deactivate": {
+    /**
+     * Deactivate a team
+     * @description Deactivate a team and stop all running workloads for a team.
+     */
+    post: operations["admin-team-deactivate"];
+  };
+  "/admin/teams/{teamId}/members": {
+    /**
+     * List team members by role
+     * @description Will return a paginated list of team members matching the supplied roles.
+     */
+    get: operations["admin-teamMemberships-list"];
+  };
+  "/admin/teams/{teamId}/reactivate": {
+    /**
+     * Reactivate a team
+     * @description Reactivate a team.
+     */
+    post: operations["admin-team-reactivate"];
+  };
+  "/admin/teams/{teamId}/suspend": {
```

Comment:
Do we want admin endpoints in the CLI? I thought those were for internal use

Reply:
Approved Fix [here](https://github.internal.digitalocean.com/paperspace/ps-api/pull/191#pullrequestreview-485444).
I assume this file needs to be updated once the other PR gets merged?